modules/adot/templates/adot-values.yaml.tpl

```diff
@@ -136,6 +114,29 @@ adotCollector:
           limit_mib: 100
           check_interval: 5s
       exporters:
+        awsemf/prometheus:
+          dimension_rollup_option: NoDimensionRollup
+          log_group_name: adot-log-group-dasmeta
+          log_stream_name: adot-metrics-prometheus
```

Comment:
let's define a local variable `log_stream_name = "adot-metrics"`, which will be used in awsemf exporter and in awsemf/prometheus exporter like this: `log_stream_name = "${log_stream_name}-prometheus"